### PR TITLE
Update README once persistent-S3-BDBags are implemented (resolves #55)

### LINKED ISSUE #55: Update README in cgp-deployment once persistent-S3-BDBags are implemented
Append to README to explain

* newly added functionality
* anwers to the newly added prompt questions during `cgp deployment` (AWS credentials, etc.)

┆Issue is synchronized with this [JIRA Story](https://ucsc-cgl.atlassian.net/browse/DEP-25)
┆Project Name: CGP-Deployment
┆Issue Number: DEP-25
┆Epic: Epic 8: “Handoff” From Portal to Workspace

### PR DESCRIPTION
**Pull request by mjkrause**
* removed all references relating to the _bagit-firecloud-lambda_
* added short explanation how to answer prompt question referring to the AWS S3 bucket (name and AWS region)
* unrelated to this story: removed line break in one header

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -224,11 +223,9 @@ what to put for each step.
    You will also be asked for a Google site verfication code. For this
    field you can just enter `NONE`.
 
-1. The bagit-firecloud-lambda is use for the **Export to Firecloud**
-   button.
-
 1. The [S3 bucket you created earlier](#makebucket) is what you should
-   use for this next step.
+   use for this next step. Provide the name you assigned to the bucket,
+   and the AWS region that bucket resides in.
```
Comment (coverbeck) [inline README.md]: ...that **the** bucket resides in.

File: README.md [hunk 2]
```diff
@@ -117,8 +117,7 @@ so Amazon is aware of it. Set the privileges of that key pair file to
 _read-by-user-only_ by `chmod 400 ~/.ssh/<your_key>.pem` so it is not
 publicly viewable.
 
-#### <a name="makebucket"></a> Create an AWS S3 bucket for persistent
-storage of BDBag
+#### <a name="makebucket"></a> Create an AWS S3 bucket for persistent storage of BDBag
```
Comment (coverbeck) [inline README.md]: ... of BDBag**s**

**Review by coverbeck — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mjkrause) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (mjkrause) [reply README.md]: fixed

**Review by coverbeck — APPROVED** [review]

**Pull request merged**